src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
```suggestion
    void draw();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/842fc418-caa3-46ec-aa36-474559a03bdd)